facade/src/main/java/org/jboss/pnc/facade/providers/BuildConfigurationProviderImpl.java

```diff
@@ -219,6 +239,9 @@ public BuildConfigurationRevision createRevision(String id, BuildConfiguration b
             return buildConfigurationRevisionMapper.toDTO(latestRevision);
         }
         bcEntity.setCreationTime(latestRevision.getCreationTime());
+        org.jboss.pnc.model.User user = userService.currentUser();
+        bcEntity.setCreationUser(user);
```

Comment:
Here you are changing the creationUser of existing BC. This method creates new BC-audited for existing BC. It's called createRevision because from API point it is creating new entity, but it's actually just updating the BC.

Reply:
ah good point, thanks for the clarification, the name tricked me, I am now going to update just the lastModificationUser